clients/client-java/src/main/java/org/apache/gravitino/client/GravitinoClient.java

```diff
@@ -308,6 +313,32 @@ public Group revokeRolesFromGroup(List<String> roles, String group)
     return getMetalake().revokeRolesFromGroup(roles, group);
   }
 
+  /**
+   * Get the owner of a metadata object.
+   *
+   * @param object The metadata object
+   * @return The owner of the metadata object. If the metadata object doesn't set the owner, it will
+   *     return Optional.empty().
+   * @throws NotFoundException If the metadata object is not found
+   * @throws NoSuchMetadataObjectException If the metadata object is not found.
+   */
+  public Optional<Owner> getOwner(MetadataObject object) throws NoSuchMetadataObjectException {
+    return getMetalake().getOwner(object);
+  }
+
+  /**
+   * Set the owner of a metadata object.
+   *
+   * @param object The metadata object.
+   * @param ownerName The name of the owner
+   * @param ownerType The type of the owner, The owner can be a user or a group.
+   * @throws NotFoundException If the metadata object isn't found or the owner doesn't exist.
```

Comment:
Should this method also need to throw `NoSuchMetadataObjectException`?

Reply:
Owner is not the metadata object. So it's suitable to use NotFoundException instead of `NoSuchMetadataObjectException`